data/pages/career-de.md

```diff
@@ -3,8 +3,4 @@ title: Karriere
 language: de
 ---
 
-Satellytes – das sind aktuell 14 ausschließlich leidenschaftliche Entwickler:innen und Designer:innen. Wir haben großen Spaß an Technologie und freuen uns auf neue Herausforderungen. Dabei fokussieren wir uns auf langfristige Engagements im Konzerngeschäft.
-
-Neuen Aufgaben begegnen wir immer mit angemessenem Respekt. Wir streben stets hochwertige und zeitgemäße Lösungen an – die Wahl der Technologie ist für uns dabei sekundär.
-
-Unser [Büro](/de/office/) befindet sich in einem wunderschönen Altbau im Herzen Münchens, in der Sendlinger Straße, unweit der gleichnamigen U-Bahn-Station. In Laufweite gibt es nicht nur dutzende Läden, Cafés, Restaurants und den bekannten Viktualienmarkt, sondern auch diverse kleinere und größere Parks. Sogar die Isar ist in der Mittagspause fußläufig oder mit der U-Bahn in nur 3 Minuten zu erreichen.
+This file is only used to match the preview card to the career page.
```

Comment:
I don't understand the relationship to the preview cards. Can't we get rid of them entirely?

Reply:
The preview card is tied to the markdown or PersonioJob type. I think we can remove this markdown file, as it doesn't give a lot of value to have a preview card for this page. The default card is fine enough. I tried to pin the preview card to the general `SitePage`, but this didn't work as we need the meta data from the markdown or personio type. I guess it would be possible to make it work, but decided to not do it now.

```ts
export const createPreviewCards = async ({
  node,
  ...rest
}: CreateNodeArgs<SatellytesNode>) => {
  if (node.internal.type === 'SyPersonioJob') {
    await createPreviewCard(node.name, { node, ...rest });
  }

  if (node.internal.type === 'MarkdownRemark') {
    await createPreviewCard(node.frontmatter.title, { node, ...rest });
  }
};
```